app/views/merchants/items/edit.html.erb

```diff
@@ -2,12 +2,12 @@
 
 <h3>Edit <%= @item.name %></h3>
 
-<%= form_with model: @item, url: merchant_item_path(@merchant.id, @item.id), method: :patch, local: true do |f| %>
+<%= form_with model: [@merchant, @item], local: true do |f| %>
```

Comment:
Are you using both the merchant & item models for this form because for each merchant.id, it will have multiple item.id's as well so you need both?

Reply:
Yea, so I think because my resources are all embedded under a merchant, it needs both ids. So my path here is like "/merchants/merchant_id/items/item_id" so to access that particular model, the form needs to know which merchant's item and also which item in particular